Gems/DiffuseProbeGrid/Code/Source/Render/DiffuseProbeGridRayTracingPass.cpp

```diff
@@ -90,17 +98,21 @@ namespace AZ
             RHI::RayTracingPipelineStateDescriptor descriptor;
             descriptor.Build()
                 ->PipelineState(m_globalPipelineState.get())
-                ->MaxPayloadSize(64)
+                ->MaxPayloadSize(96)
                 ->MaxAttributeSize(32)
-                ->MaxRecursionDepth(2)
+                ->MaxRecursionDepth(16)
```

Comment:
Shouldnt this be in a cvar or settings registry instead of hardcoded here?

Reply:
It depends on the usage in the shader, this change is specific to the DiffuseProbeGrid ray tracing pass, other passes may want more or less recursion.